tests/models/llava/test_modeling_llava.py

```diff
@@ -186,6 +184,188 @@ def test_training_gradient_checkpointing_use_reentrant(self):
     def test_training_gradient_checkpointing_use_reentrant_false(self):
         pass
 
+    # Copied from tests.test_modeling_common.ModelTesterMixin.test_resize_tokens_embeddings with config.vocab_size->config.text_config.vocab_size
```

Comment:
I've tried that, but that doesn't work. The test fails with:
```python
# Check that resizing the token embeddings with a larger vocab size increases the model's vocab size
            model_embed = model.resize_token_embeddings(model_vocab_size + 10)
>           self.assertEqual(model.config.vocab_size, model_vocab_size + 10)
E           AssertionError: 99 != 109
```
This is because when calling the resize_token_embeddings method, we are changing `config.text_config.vocab_size`, whereas `config.vocab_size` remains unchanged.

Reply:
That's because you removed the line that properly updates the config value in `resize_token_embeddings`

Reply:
Yes, I agree we don't want to just add stuff to make the tests pass. I mistakenly though `vocab_size` was still being set in the config for BC, in which case, we would have still needed to update the parameter. 

If we leave as a property to deprecate (cf notes in vip llava config file) then we can leave copied from headers here